ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.13.yaml

```diff
@@ -0,0 +1,50 @@
+base_images:
+  ansible:
+    name: "4.13"
+    namespace: ocp
+    tag: ansible
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: openstack-installer
+  upi-installer:
+    name: "4.13"
+    namespace: ocp
+    tag: upi-installer
+releases:
+  initial:
+    candidate:
+      product: ocp
+      relative: 1
+      stream: nightly
+      version: "4.13"
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: installer-rehearse-ibmcloud
+  interval: 960h
+  steps:
+    cluster_profile: ibmcloud-qe
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi
+- as: installer-rehearse-ibmcloud-private
```

Comment:
Generally we only need one rehearse job for each platform, so I guess you do not need this test item.

Reply:
how about just keep installer-rehearse-ibmcloud-private in rehearse job?

Reply:
Then you need to replace `workflow: cucushift-installer-rehearse-ibmcloud-ipi` with `workflow: cucushift-installer-rehearse-ibmcloud-ipi-private` in the above test item.

Reply:
Suggest to rename the test name to `installer-rehearse-ibmcloud`, just like what I explained, we are using this test job to validate the developing PR, not used of our regular ci jobs, so one platform only need one test item.

Reply:
1. is it OK?  use workflow cucushift-installer-rehearse-ibmcloud-ipi-private as installer-rehearse-ibmcloud ? 
> - as: installer-rehearse-ibmcloud
>   interval: 960h
>   steps:
>     cluster_profile: ibmcloud-qe
>     workflow: cucushift-installer-rehearse-ibmcloud-ipi-private
2. whether the in the openshift-verification-tests-master__installer-rehearse-*.yaml , different version check different workflow, the name are all  same as` 'installer-rehearse-ibmcloud'` ?

Reply:
For 1, any concern?
For 2, I explained that in the above comments, the `'installer-rehearse-ibmcloud` is only used for testing developing PR purpose, it does not show up in our regular ci failure watch list.